xml/HumanitarianClusters.xml

```diff
@@ -0,0 +1,194 @@
+<codelist xmlns:budget-alignment="http://iatistandard.org/activity-standard/overview/country-budget-alignment/" name="Humanitarian Clusters" xml:lang="en" category-codelist="HumanitarianClusters" complete="1" embedded="0">
+    <metadata>
+        <name>
+            <narrative>Global Coordination Groups</narrative>
+            <narrative xml:lang="fr">Secteur du CAD à 5 chiffres</narrative>
+        </name>
+        <url>https://data.humdata.org/dataset/global-coordination-groups-beta</url>
+        <category>
+            <narrative>CodeforIATImaintained</narrative>
+        </category>
+    </metadata>
+    <codelist-items>
+        <codelist-item status="active">
+            <code>4</code>
+            <name>
+                <narrative>Emergency Shelter and NFI</narrative>
+                <narrative xml:lang="fr">Abris d'urgence et NFI</narrative>
+            </name>
+            <description>
+                <narrative>Shelter and Non-Food Items includes provision of shelter materials and non-food household item packages. The theme also covers Camp Coordination and Camp Management. Long-term/permanent reconstruction/rebuilding of housing is covered under "Recovery and Reconstruction."</narrative>
```

Comment:
I’m not 100% sure about creating unofficial descriptions like this…

Reply:
Yes, me neither, do you know if the description element is required? Or we could duplicate the Narrative